core/resources/intent_example.xml

```diff
@@ -0,0 +1,83 @@
+<h:html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:orx="http://openrosa.org/jr/xforms" xmlns="http://www.w3.org/2002/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa" xmlns:vellum="http://commcarehq.org/xforms/vellum">
```

Comment:
Is this doc used anywhere? 

Might be kind of hard to actually test the inputs/outputs automatically, just checking whether we had a clear meaning for this folder.